src/SwedbankPay.Sdk/Payments/SwishPayments/SwishPayment.cs

```diff
@@ -1,14 +1,16 @@
-﻿using System;
+﻿using SwedbankPay.Sdk.Extensions;
+using System;
+using System.Net.Http;
 using System.Threading.Tasks;
 
-namespace SwedbankPay.Sdk.Payments.Swish
+namespace SwedbankPay.Sdk.Payments.SwishPayments
 {
-    public class Payment
+    public class SwishPayment
     {
-        private Payment(PaymentResponse paymentResponse, SwedbankPayHttpClient client)
+        private SwishPayment(SwishPaymentPaymentResponse paymentResponse, HttpClient client)
         {
             PaymentResponse = paymentResponse.Payment;
-            var operations = new Operations();
+            var operations = new SwishPaymentOperations();
 
             foreach (var httpOperation in paymentResponse.Operations)
```

Comment:
This loop should be moved into the constructor of `SwishPaymentOperations` for [SRP](https://en.wikipedia.org/wiki/Single_responsibility_principle) and immutability.